Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Material/MaterialComponentBus.h

```diff
@@ -95,8 +99,16 @@ namespace AZ
             virtual void ClearPropertyOverrides(const MaterialAssignmentId& materialAssignmentId) = 0;
             //! Clear all property overrides
             virtual void ClearAllPropertyOverrides() = 0;
+            //! Set Property overrides for a specific material assignment
+            virtual void SetPropertyOverrides(
+                const MaterialAssignmentId& materialAssignmentId, const MaterialPropertyOverrideMap& propertyOverrides) = 0;
             //! Get Property overrides for a specific material assignment
             virtual MaterialPropertyOverrideMap GetPropertyOverrides(const MaterialAssignmentId& materialAssignmentId) const = 0;
+            //! Set Model UV overrides for a specific material assignment
+            virtual void SetModelUvOverrides(
+                const MaterialAssignmentId& materialAssignmentId, const AZ::RPI::MaterialModelUvOverrideMap& modelUvOverrides) = 0;
+            //! Get Model UV overrides for a specific material assignment
+            virtual AZ::RPI::MaterialModelUvOverrideMap GetModelUvOverrides(const MaterialAssignmentId& materialAssignmentId) const = 0;
```

Comment:
should this be 
`const  AZ::RPI::MaterialModelUvOverrideMap&`

Reply:
No. I cannot guarantee that the implementation is storing a local container or has an entry for it. I suppose I could return a dummy reference if one was not found.